app/src/main/java/com/example/shoppingcalculator/LoginActivity.kt

```diff
@@ -23,7 +23,8 @@ class LoginActivity: AppCompatActivity() {
         setContentView(R.layout.activity_login)
 
         var fing = VKUtils.getCertificateFingerprint(this, this.packageName)
-        if (fing != null) {
+        if (fing != null          )
+        {
```

Comment:
🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (8) (should be 12)